core/templates/dev/head/pages/creator_dashboard/creator_dashboard.html

```diff
@@ -60,19 +60,15 @@ <h1 class="stat-value"><[subscribersList.length]></h1>
              translate="I18N_DASHBOARD_EXPLORATIONS">
           </a>
         </li>
-        {% if not can_create_collections %}
-          <!-- A layout fix when the 'My Collections' tab is absent. -->
-          <li ng-if="collectionsList.length === 0 && subscribersList.length === 0"></li>
-        {% endif %}
-        {% if can_create_collections %}
-          <li ng-if="collectionsList.length > 0"
-              ng-class="{'oppia-dashboard-tabs-active': activeTab === 'myCollections'}">
-            <a class="oppia-dashboard-tabs-text"
-               ng-click="setActiveTab('myCollections')"
-               translate="I18N_DASHBOARD_COLLECTIONS">
-            </a>
-          </li>
-        {% endif %}
+        <!-- A layout fix when the 'My Collections' tab is absent. -->
+        <li ng-if="collectionsList.length === 0 && subscribersList.length === 0 && !canCreateCollections"></li>
```

Comment:
I discussed it with @shubha1593 and in her words
`we should check whether user can create collection or not first, so just place that condition first i.e. ng-if="!canCreateCollections && collectionsList.length === 0 && subscribersList.length === 0".` 
and I'm too agree with this.

Reply:
Ok, done.